Add Research Papers that contains DVC tool (#12)
iterative/awesome-iterative-projects: readme.md

* [ModelOriented/MAIR](https://github.com/ModelOriented/MAIR): Monitoring impact of AI regulations with a DVC pipeline.
* [Kaggle-Titanic-DVC](https://dagshub.com/kingabzpro/kaggle-titanic-dvc): Survival analysis DVC experiment.
* [VQA-With-Multimodal-Transformers](https://github.com/tezansahu/VQA-With-Multimodal-Transformers): Visual Question Answering task on the [DAQUAR Dataset](https://www.kaggle.com/tezansahu/processed-daquar-dataset) using multimodal [transformer models](https://huggingface.co/docs/transformers/index) with an experiment pipeline tracked in DVC Studio.

> ## Iterative Tools in Research Papers 

Reviewer: Looks great! A few papers aren't directly focusing on Iterative tools anywhere in them, so it might be better to either remove those or reword this section title.

Reviewer: Sorry @mertbozkir, I see this name came from @jendefig! @jendefig What do you think?

Reviewer 2: We will adjust and move to another repo we will be creating for UGC.

* [On the Co-evolution of ML Pipelines and Source Code](https://mcis.cs.queensu.ca/publications/2021/saner.pdf)  - Empirical Study of DVC Projects.

> * [FAIRly big](https://www.nature.com/articles/s41597-022-01163-2) -  A framework for computationally reproducible processing of large-scale data.

Reviewer: This one is more of an alternative that mentions DVC briefly as a point of comparison.

> * [Discovering temporal scientometric knowledge in COVID-19 scholarly production](https://paperity.org/p/281730053/discovering-temporal-scientometric-knowledge-in-covid-19-scholarly-production)

Reviewer 3: ```suggestion
* [Discovering temporal scientometric knowledge in COVID-19 scholarly production](https://doi.org/10.1007/s11192-021-04260-y): Using DVC to explore 200k published articles
```

> * [YMIR](https://deepai.org/publication/ymir-a-rapid-data-centric-development-platform-for-vision-applications) - A Rapid Data-centric Development Platform for Vision Applications.

Reviewer 3: ```suggestion
* [YMIR](https://arxiv.org/pdf/2111.10046): A Rapid Data-centric Development Platform for Vision Applications.
```

isn't it better to link to https://github.com/IndustryEssentials/ymir instead though?

```suggestion
```

> * [DevOps in Academic Research](https://mattsegal.dev/devops-academic-research.html) - Compherensive research about DevOps including MLOps.

Reviewer: This one recommends looking at DVC but uses Git LFS instead. Maybe not worth including?

> * [DevOps in Academic Research](https://mattsegal.dev/devops-academic-research.html) - Compherensive research about DevOps including MLOps.

Reviewer 3: doesn't actually use Iterative tools?

```suggestion
```

> * [Versioning Data Is About More than Revisions](https://datascience.codata.org/articles/10.5334/dsj-2021-012/) - A Conceptual Framework and Proposed Principles.

Reviewer: Interesting and relevant, although I don't see that it references any Iterative tools 🤔 .

> * [Versioning Data Is About More than Revisions](https://datascience.codata.org/articles/10.5334/dsj-2021-012/) - A Conceptual Framework and Proposed Principles.

Reviewer 3: doesn't actually use Iterative tools?

```suggestion
```

> * [Asset Management in Machine Learning](https://arxiv.org/pdf/2102.06919.pdf) - A survey about practical MLOps techniques and tools.

Reviewer 3: ```suggestion
* [Asset Management in Machine Learning](https://arxiv.org/pdf/2102.06919): A summary of 17 ML management tools for data & experiments.
```

not actually sure about this article; doesn't really do a comparison nor does it have any particularly strong conclusion.

It also mistakenly doesn't mention DVC on page 9 in the "Implicit Collection" para (DVC doesn't need you to alter source code) and the "External Validity" para (DVC doesn't assume you use Python)

```suggestion
```
